packages/fetch-http-handler/src/stream-collector.ts

```diff
@@ -1,21 +1,14 @@
 import { StreamCollector } from "@smithy/types";
-import { fromBase64 } from "@smithy/util-base64";
 
 //reference: https://snack.expo.io/r1JCSWRGU
-export const streamCollector: StreamCollector = (stream: Blob | ReadableStream): Promise<Uint8Array> => {
+export const streamCollector: StreamCollector = async (stream: Blob | ReadableStream): Promise<Uint8Array> => {
   if (typeof Blob === "function" && stream instanceof Blob) {
-    return collectBlob(stream);
+    return new Uint8Array(await stream.arrayBuffer());
```

Comment:
could you update the unit tests for this?

what kind of performance diff does this bring?